### PR TITLE
kldiv loss

### PR DESCRIPTION
**Pull request by szha**
this is an elementwise loss for regression tasks

### CODE REVIEW

File: python/mxnet/gluon/loss.py [hunk 1]
```diff
@@ -171,3 +171,43 @@ def hybrid_forward(self, F, output, label, sample_weight=None):
...
+    from_logits : bool, default False
+        whether input is log probability (usually from log_softmax) instead
+        of unnormalized numbers.
+    weight : float or None
+        global scalar weight for loss
+    sample_weight : Symbol or None
+        per sample weighting. Must be broadcastable to
+        the same shape as loss. For example, if loss has
+        shape (64, 10) and you want to weight each sample
+        in the batch, sample_weight should have shape (64, 1)
```
Comment (piiswrong) [inline python/mxnet/gluon/loss.py]: add doc for batch_axis

File: python/mxnet/gluon/loss.py [hunk 2]
```diff
@@ -171,3 +171,43 @@ def hybrid_forward(self, F, output, label, sample_weight=None):
...
+
+    .. _Kullback-Leibler divergence:
+        https://en.wikipedia.org/wiki/Kullback-Leibler_divergence
+    .. math::
+        L = 1/n \\sum_i (label_i * (log(label_i) - output_i))
+    label's shape should be the same as output's.
+
+    Parameters
+    ----------
+    from_logits : bool, default False
```
Comment (piiswrong) [inline python/mxnet/gluon/loss.py]: default True

File: python/mxnet/gluon/loss.py (hunk 1 above)
Comment (szha) [inline python/mxnet/gluon/loss.py]: added
